### ISSUE TITLE
WPA2 Enterprise wifi fault

### ISSUE THREAD

**Issue by zakhaj** [BenSova/Patched-Sur#134]
A1425 Late 2012 MBPr 13"

Anyone else having problems with WPA2 enterprise wifi certificates? I can no longer connect to any workplace wifi's which issue a security certificate, 802.11X.  I get the popup giving details of the certificate, click continue, then it connects but will not assign an IP therefore self assigns an IP, and therefore wont connect to the internet. I tried DHCP new release etc, but to no avail. This is happening on all WPA2 EAP Enterprise networks...

WPA2 personal and unsecured/public networks are fine, so my wifi at home is perfect.

I've seen the same issue on another discussion page however there was no reply.

I used 0.04beta release install.

everything else is working perfect..... ps I had no issues on Catalina.

@BenSova 

thanks for any guidance.

**Comment by zakhaj** [comment]
furthermore.... if I replaced my stock airport card (BCM94331CSAX) with a late 2013 ac capable card (BCM94360CSAX) - would that solve the issue above? or not?

thanks 

ps. credit to all who made this patcher and all the supporting pieces, BarryKN etc. great stuff.

**Comment by zakhaj** [comment]
Can anyone help or shed any light?

**Comment by gservera** [comment]
Same problem here on Mid 2012 MBPr 15"

**Comment by zakhaj** [comment]
> Same problem here on Mid 2012 MBPr 15"

not just me then...!

**Comment by lokolor** [comment]
I face the same problem

**Comment by zakhaj** [comment]
Replace your wifi card with the late 2013 model. (If youre using a 2012 mbpr)

**Comment by ososx** [comment]
Having the same problem on my 2013 early MBP - all WiFi connections work BUT the WPA2 enterprise. BenSova wrote that this is a well known issue. I know of no fix at this time.

**Comment by TheMac6ine** [comment]
Hallo together, 

I upgraded my MBP early 2013 to Big Sur and I have no problems with WPA2 Enterprise (PEAP not TLS) 

But I also updated my Wifi card to the ac one.

**Comment by andyhone** [comment]
While connecting to WPA2 Enterprise, I found that the 802.1X in th network preference window shows that the authenticated is ticking. But my administrator said that the authentication is done but Mac can not get IP address from DHCP after the DHCP has assigned to.

**Comment by southerndevz** [comment]
Does this problem persist if I buy an external USB WiFi dongle? I need Big Sur but also at work we use WPA2 Enteprise

**Comment by Brin-o** [comment]
Hi! Currenlty facing the same issue here on a 2012 15" Retina (MacBookPro10,1). Can anyone confirm if swapping the wifi card fixes this and if so what model should I swap it to?

**Comment by zakhaj** [comment]
Yes it does fix the problem. Very easy. Look on Amazon or eBay or similar for a BCM94360CSAX

(Late 2013 WiFi ac capable card).
